index.html

```diff
@@ -314,18 +318,18 @@ <h4>Multikey</h4>
           </pre>
 
           <p>
-            The `secretKeyMultibase` property represents a Multibase-encoded Multikey
-            expression of an Ed25519 secret key (sometimes also referred to as a private key).
-            The value starts with the two-byte prefix `0x8026` (the varint expression of `0x1300`),
-            followed by the 32-byte Ed25519 secret key data. The combined 34-byte
-            value is then base58-btc encoded and `z` is added as the prefix on the
-            encoded value.
+The `secretKeyMultibase` property represents a Multibase-encoded Multikey
+expression of an Ed25519 secret key (sometimes also referred to as a private
+key). The value starts with the two-byte prefix `0x8026`,
```

Comment:
Again, these prefixes need to be normatively defined.

Reply:
The Multikey prefixes for EdDSA public and private keys are defined here: https://pr-preview.s3.amazonaws.com/w3c/vc-di-eddsa/pull/63.html#multikey